src/Chunk.h

```diff
@@ -95,7 +95,7 @@ class cChunk :
 	/** Marks all clients attached to this chunk as wanting this chunk. Also sets presence to cpQueued. */
 	void MarkRegenerating(void);
 
-	/** Returns true iff the chunk has changed since it was last saved. */
+	/** Returns true if the chunk has changed since it was last saved. */
```

Comment:
This is not a typo, it is a thing normally used in maths and logic. It means `if and only if`.